packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/feature_login/views/login_page.dart

```diff
@@ -44,22 +48,22 @@ class LoginPage extends StatelessWidget {
                         mainAxisSize: MainAxisSize.max,
                         mainAxisAlignment: MainAxisAlignment.spaceBetween,
                         children: [
-                          Expanded(
-                            flex: 1,
-                            child: Divider(
-                                color: context.designSystem.colors.gray),
-                          ),
-                          Padding(
-                            padding: EdgeInsets.symmetric(
-                                horizontal: context.designSystem.spacing.xs),
-                            child: Text(context.l10n.featureLogin.or),
-                          ),
-                          Expanded(
-                            flex: 1,
-                            child: Divider(
-                                color: context.designSystem.colors.gray),
-                          ),
-                        ],
+                            Expanded(
+                              flex: 1,
+                              child: Divider(
+                              color: context.designSystem.colors.gray),
+                            ),
+                            Padding(
+                              padding: EdgeInsets.symmetric(
+                              horizontal: context.designSystem.spacing.xs),
+                              child: Text(context.l10n.featureLogin.or),
+                            ),
+                            Expanded(
+                              flex: 1,
+                              child: Divider(
+                              color: context.designSystem.colors.gray),
```

Comment:
As mentioned on the other comment the design system should not refer to the actual colors, since it will change based on the light/dark mode as well as if we need to rebrand the application.

Reply:
This isn't my code, and I've discovered we have a bunch of places where we're using this `gray` color:
<img width="708" alt="Screenshot 2024-12-23 at 09 58 14" src="https://github.com/user-attachments/assets/cc58e89c-27a7-44bb-998d-4ffff61b4045" />

Changing all of this is a bit outside the scope of this PR. In any case, I've added the `dividerColor` to designSystem for now and used it on LoginPage.